docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplateBase.java

```diff
@@ -279,6 +295,10 @@ public CreateContainerCmd fillContainerConfig(CreateContainerCmd containerConfig
         if (environment != null && environment.length > 0)
             containerConfig.withEnv(environment);
 
+        if (getMacAddress() != null && !getMacAddress().trim().isEmpty()) {
```

Comment:
its apache commons-lang, not guava. Jenkins's Utils.* mostly copy-paste from there